frameworks/hdfs/universe/config.json

```diff
@@ -13,13 +13,18 @@
         "user": {
           "type": "string",
           "description": "The linux user used to run the scheduler and all executors.",
-          "default": "root"
+          "default": "nobody"
         },
         "principal" : {
           "description":"The principal for the HDFS service instance.",
           "type":"string",
           "default":""
         },
+        "mesos_api_version": {
```

Comment:
OK per conversation elsewhere this option will likely be removed from everything except hello-world soon. Can just add it here for now as a no-op for CI compatibility, and then remove it again in a later pass.